content/tutorials/manage-the-rippled-server/configuration/run-rippled-in-reporting-mode.md

```diff
@@ -0,0 +1,310 @@
+2021-Dec-09 21:31:53.285651 UTC Amendments:DBG Amendment 452F5906C46D46F407883344BFDD90E672B672C5E9943DB4891E3A34FEEEB9DB (fixSTAmountCanonicalize) is supported and will be up voted if not enabled on the ledger.
+2021-Dec-09 21:31:53.285725 UTC Amendments:DBG Amendment 2CD5286D8D687E98B41102BDD797198E81EA41DF7BD104E6561FEB104EFF2561 (fixTakerDryOfferRemoval) is supported and will be up voted if not enabled on the ledger.
+2021-Dec-09 21:31:53.290446 UTC Server:NFO Opened 'port_rpc_admin_local' (ip=127.0.0.1:7005, admin IPs:127.0.0.1, http)
+2021-Dec-09 21:31:53.290834 UTC Server:NFO Opened 'port_ws_admin_local' (ip=127.0.0.1:7006, admin IPs:127.0.0.1, ws)
+2021-Dec-09 21:31:53.290984 UTC Application:WRN Running in standalone mode
+2021-Dec-09 21:31:53.291048 UTC NetworkOPs:NFO STATE->full
+2021-Dec-09 21:31:53.291192 UTC Application:FTL Startup RPC: 
+{
+	"command" : "log_level",
+	"severity" : "debug"
+}
+
+
+2021-Dec-09 21:31:53.291347 UTC RPCHandler:DBG RPC call log_level completed in 2.2e-08seconds
+2021-Dec-09 21:31:53.291440 UTC Application:FTL Result: 
+{
+	"warnings" : 
+	[
+		
+		{
+			"id" : 1004,
+			"message" : "This is a reporting server.  The default behavior of a reporting server is to only return validated data. If you are looking for not yet validated data, include \"ledger_index : current\" in your request, which will cause this server to forward the request to a p2p node. If the forward is successful the response will include \"forwarded\" : \"true\""
+		}
+	]
+}
+
+
+2021-Dec-09 21:31:53.291502 UTC ReportingETL:NFO Starting reporting etl
+2021-Dec-09 21:31:53.291605 UTC Application:NFO Application starting. Version is 1.8.1+DEBUG
+2021-Dec-09 21:31:53.291747 UTC LoadManager:DBG Starting
+2021-Dec-09 21:31:53.291846 UTC gRPC Server:NFO Starting gRPC server at 0.0.0.0:60051
+2021-Dec-09 21:31:53.293246 UTC LedgerCleaner:DBG Started
+2021-Dec-09 21:31:53.295543 UTC ReportingETL::ETLSource:DBG handleMessage : Received a message on ledger  subscription stream. Message : {
+   "result" : {},
+   "status" : "success",
+   "type" : "response"
+}
+ - { validated_ledger :  , ip : 127.0.0.1 , web socket port : 6006, grpc port : 50051 }
+2021-Dec-09 21:31:53.368075 UTC ReportingETL:NFO monitor : Database is empty. Will download a ledger from the network.
+2021-Dec-09 21:31:53.368183 UTC ReportingETL:NFO monitor : Waiting for next ledger to be validated by network...
+```
+
+## Frequently Asked Questions
+
+### Do I need to run more than one instance of rippled in order to use Reporting mode?
+
+Yes. A rippled node running in reporting mode does not connect to the P2P network, but instead extracts validated data from one or more rippled nodes that are connected to the P2P network. In addition to the rippled node running in reporting mode, you need to run at least one other rippled node that is connected to the P2P network. 
```

Comment:
Mention that you need to trust the P2P mode server, since a malicious one could feed you wrong data, so that's why you have to run both yourself.